packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -446,9 +446,10 @@ const schema: SchemaType<"Posts"> = {
   }),
 
   pageUrl: resolverOnlyField({
-    type: String,
+    type: 'String',
+    graphQLtype: 'String!',
     canRead: ['guests'],
-    resolver: (post: DbPost, args: void, context: ResolverContext) => postGetPageUrl(post, true),
+    resolver: (post: DbPost, args: void, context: ResolverContext) : string => postGetPageUrl(post, true),
```

Comment:
nit: no space between closing parentheses and the `:` that starts the return type annotation